spec/keys_spec.js

```diff
@@ -0,0 +1,8 @@
+import { keys } from '../dist/keys.js'
+
+describe( 'keys', () => {
+  it( 'retuns keys in an object', () => {
+    expect(keys({'a':1,'c':3,'b':2}) ).toEqual(['a','c','b'])
+  })
```

Comment:
What happens when an empty object is supplied? What happens if a non object is supplied (like an array? int?)